prov/efa/src/rxr/rxr_pkt_type_data.c

```diff
@@ -147,7 +144,7 @@ ssize_t rxr_pkt_send_data_mr_cache(struct rxr_ep *ep,
 	/* Assign packet header in constructed iov */
 	iov[i].iov_base = rxr_pkt_start(pkt_entry);
 	iov[i].iov_len = sizeof(struct rxr_data_hdr);
-	desc[i] = rxr_ep_mr_local(ep) ? fi_mr_desc(pkt_entry->mr) : NULL;
```

Comment:
This should probably go in a separate PR or another patch in this series where we just remove `rxr_ep_mr_local` completely.

Reply:
Make sense, reverted.